### ISSUE TITLE
WSL ask be to trust every workspace agian, even those I had open before.

### ISSUE THREAD

**Issue by dbaeumer** [microsoft/vscode-remote-release#8070]
<img width="646" alt="image" src="https://user-images.githubusercontent.com/1931590/220134350-74ba876a-4dec-4852-96c0-c8741ccd64db.png">

Looks like the URL of the workspace changed.

**Comment by dbaeumer** [comment]
This only seems to happen when opening a folder from a WSl terminal using `code-insider .`

**Comment by aeschli** [comment]
@dbaeumer This looks like the folder was opened in the Windows file system (not in WSL).
I tried to repro with the latest WSL extension, and latest insiders. but it works for me

Make sure the WSL extension is installed.

If it is, can you run
`VSCODE_WSL_DEBUG_INFO=true code-insiders .`  and paste the output here?

**Comment by dbaeumer** [comment]
I am pretty sure that I never go on the Linux file system from a Windows prompt. Only from Linux. I will keep an eye on it. OK for me to close for now since I have not seen it yesterday.

**Comment by aeschli** [comment]
Yes, it's a bug with the script that is runs on Linux. The command line I was pasting is for running in a Linux terminal

I suspect there was a problem finding the WSL extension. I've seen similar reports but it seems to be one-offs.
@sandy081 Was there a problem with `--locate-extension` not always working?

**Comment by sandy081** [comment]
I do not think so, I just tried locally and it worked. I think this is not support for custom profiles, in case WSL extension is installed in custom profile.

**Comment by dbaeumer** [comment]
I saw this with my default profile.

**Comment by lszomoru** [comment]
@dbaeumer, this one felt off my radar, sorry about that. Are you still able to reproduce this issue?
